pkg/cluster/node.go

```diff
@@ -209,7 +238,7 @@ func tryUntil(until time.Time, try func() bool) bool {
 // LoadImages loads image tarballs stored on the node into docker on the node
 func (nh *nodeHandle) LoadImages() {
 	// load images cached on the node into docker
-	if err := nh.Run(
+	if err := nh.RunQ(
```

Comment:
It'd be awesome if we can make `--loglevel=debug` also print all of these messages, for debug purposes.

From what I can tell, with this patch there will no longer be any way to debug failures.

Also, if a particular step fails can we make it print out the stdout/stderr of that failed command?

Reply:
I was thinking maybe `--loglevel=trace` should do this after we upgrade logrus, but debug actually sounds good.

Definitely we should print out failed command output. Especially for kubeadm.

Reply:
I think what we really want is:
- when critical things fail, inform the user and tell them what they can do about it
- if the log level is cranked high enough, log even "silent" command output on exit